sql/src/test/java/org/opensearch/sql/sql/antlr/SQLParserTest.java

```diff
@@ -1,3 +1,9 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
```

Comment:
this seems unrelated, btw.  It's done on purpose, right?

Reply:
It was done on purpose, I do realize that is unrelated and I probably should have made a separate PR for it, but I noticed it was missing the header when I was adding the header to the file I had created.